gui/mupdf.xml.template

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<interface xmlns="http://zero-install.sourceforge.net/2004/injector/interface" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface https://docs.0install.net/specifications/feed.xsd http://0install.de/schema/desktop-integration/capabilities https://docs.0install.net/specifications/capabilities.xsd">
+  <name>mupdf</name>
+  <summary xml:lang="en">MuPDF is a lightweight PDF, XPS, and E-book viewer.</summary>
+  <description xml:lang="en"> The viewer is small, fast, yet complete. It supports many document formats, such as PDF, XPS, OpenXPS, CBZ, EPUB, and FictionBook 2. You can annotate PDF documents and fill out forms with the mobile viewers (this feature is coming soon to the desktop viewer as well).
+
+The command line tools allow you to annotate, edit, and convert documents to other formats such as HTML, SVG, PDF, and CBZ. You can also write scripts to manipulate documents using Javascript. </description>
+  <icon href="https://raw.githubusercontent.com/0install/apps/master/gui/mupdf.ico" type="image/vnd.microsoft.icon"/>
+  <icon href="https://raw.githubusercontent.com/0install/apps/master/gui/mupdf.png" type="image/png"/>
+  <category>Utility</category>
+  <homepage>https://mupdf.com/</homepage>
```

Comment:
This line needs to be added to associate the template with the feed:
```xml
  <feed-for interface="https://apps.0install.net/mupdf.xml"/>
```

Reply:
done